src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -224,6 +225,18 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     options_list=['--object-id', '-i'],
     help='The unique ID of the Azure AD administrator.')
 
+read_scale_param_type = CLIArgumentType(
+    options_list=['--read-scale'],
+    help='If enabled, connections that have application intent set to readonly '
+    'in their connection string may be routed to a readonly secondary replica. '
+    'This property is only settable for Premium and Business Critical databases.',
+    arg_type=get_enum_type(DatabaseReadScale))
+
+read_replicas_param_type = CLIArgumentType(
```

Comment:
What is the type for this parameter? int or str?

Reply:
In the SDK it is an int.  I can add that int type here, but it looks like we have a lot of other int parameters being treated as string.  Not sure what value it is to have only this one parameter defined as int.

Reply:
Fixed.